scheduler/cups-driverd.cxx

```diff
@@ -2914,3 +2927,74 @@ regex_string(const char *s)		/* I - String to compare */
 
   return (NULL);
 }
+
+void *
+cupsFileGetsCall(void *input)
+{
+  retval = cupsFileGets(((struct args*)input)->fp,((struct args*)input)->buf,((struct args*)input)->buflen);
```

Comment:
@pranjanpr you can do a following:

1) if you defined your struct with typedef before, you can use 'some_type_t' instead of writing ```struct args```
2) you can do a recast before calling ```cupsFileGets()``` into a local variable and then use a local variable for arguments. IMO it will look better than three recasts in arguments.